capacity4more/modules/c4m/features/c4m_features_homepage/plugins/content_types/video_embed/video-embed.tpl.php

```diff
@@ -0,0 +1,31 @@
+<script type="text/javascript">
```

Comment:
@nader77 You can pass it in the same way like in angular - through the settings.